engine/src/mode_development.cpp

```diff
@@ -731,7 +731,7 @@ bool MCModeHandleMessageBoxChanged(MCExecContext& ctxt, MCStringRef p_string)
 			else
 				t_msg_stack -> raise();
 
-			((MCField *)MCmessageboxredirect) -> settext(0, p_string, False);
+			((MCField *)MCmessageboxredirect -> Get()) -> settext(0, p_string, False);
```

Comment:
'openrect' might have called script, so MCmessageboxredirect -> Get() *might* return nil at this point